mlrun/api/db/sqldb/models/models_sqlite.py

```diff
@@ -158,6 +159,40 @@ class Artifact(Base, mlrun.utils.db.HasStruct):
         def get_identifier_string(self) -> str:
             return f"{self.project}/{self.key}/{self.uid}"
 
+    class ArtifactV2(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = "artifacts_v2"
+        __table_args__ = (
+            UniqueConstraint("uid", "project", "key", name="_artifacts_uc"),
+        )
+
+        Label = make_label(__tablename__)
+        Tag = make_tag_v2(__tablename__)
+
+        id = Column(Integer, primary_key=True)
+        key = Column(String(255, collation=SQLCollationUtil.collation()))
+        project = Column(String(255, collation=SQLCollationUtil.collation()))
+        kind = Column(String(255, collation=SQLCollationUtil.collation()))
+        producer_id = Column(String(255, collation=SQLCollationUtil.collation()))
+        iter = Column(Integer)
+        uid = Column(String(255, collation=SQLCollationUtil.collation()))
+        created = Column(TIMESTAMP, default=datetime.now(timezone.utc))
+        updated = Column(TIMESTAMP, default=datetime.now(timezone.utc))
+        _full_object = Column("object", BLOB)
+
+        labels = relationship(Label, cascade="all, delete-orphan")
```

Comment:
missing here:
```
tags = relationship(Tag, cascade="all, delete-orphan")
```

Reply:
I removed it after talking with @theSaarco, as some of the other tables that use tags don't have this relationship.
Is it really necessary? if so - why?

Reply:
I think it is, otherwise the cascade isn't set properly, and deleting the artifact will leave dangling Tags